### ISSUE TITLE
No longer able to login to Authentik. Failed to Authenticate

### ISSUE THREAD

**Issue by ZerkerEOD** [goauthentik/authentik#5741]
**Describe the bug**
I am getting failed to authenticate every time I put in a username and select login. It doesn't even ask for a password. The logs don't really show anything either and I can't determine where the issue is. It doesn't look like it is reaching out to my Postgres 14, or Redis servers, both respond with waiting for a connection. 

**To Reproduce**
Go to login screen and type in my username and hit login.

**Expected behavior**
Be allowed to login

**Screenshots**
![image](https://github.com/goauthentik/authentik/assets/26573385/fc41e5cc-4e98-4762-ac78-6c2298fff1bf)

**Logs**
Output of docker-compose logs or kubectl logs respectively
[authentik-2023-05-24T13-39-09.log](https://github.com/goauthentik/authentik/files/11555223/authentik-2023-05-24T13-39-09.log)

**Version and Deployment (please complete the following information):**

-   authentik version: latest - Should be 2023.5
-   Deployment: Unraid docker app

**Additional context**
I moved my app data off one share and then back into the same share to switch the drive it was on. All my other dockers came right back up except Plex, I needed to reclaim it. However, Authentik doesn't appear to work at all and it doesn't look like it is reaching out to the databases to look for the logins. I am stuck.

**Comment by BeryJu** [comment]
You can use https://goauthentik.io/docs/troubleshooting/login to re-gain access and to get more information

Also FYI you for Uptime-Kuma, you want to set the healthcheck URL to `/-/health/ready/` or `/-/health/live/` instead of just `/`, as those endpoints are dedicated for healthchecking

**Comment by ZerkerEOD** [comment]
I will look at swapping the health check for uptime-kuma. The link you sent doesn't work for me. I can console straight to the authentik docker and it says the create_recovery_key is not in PATH.

**Comment by ZerkerEOD** [comment]
Okay scratch that, I was able to get the code to work. Now my issue is that it looks like it is a fresh instance with no users, no outpost, my oath2 setup is gone. Any tips?

**Comment by ZerkerEOD** [comment]
> You can use https://goauthentik.io/docs/troubleshooting/login to re-gain access and to get more information
> 
> Also FYI you for Uptime-Kuma, you want to set the healthcheck URL to `/-/health/ready/` or `/-/health/live/` instead of just `/`, as those endpoints are dedicated for healthchecking

Okay, getting me on I was able to see an error in the logs from the system that the worker wasn't connecting to the DB. From there I searched the error on here and found someone else that had the same issue with unraid moving appdata. Looks like when they moved it, the data was no longer mapped to /mnt/cache/appdata but /mnt/user/appdata/. So I changed my pathing in Postgres and everything worked out. Thanks for the help!

**Comment by ZerkerEOD** [comment]
So, this has happened again with different issues. I can no longer log in anymore again. Looking it looks like my Postgresql container won't start, and I found this https://github.com/docker/for-mac/issues/6270. Doing what they say by removing the data from the internal mount point, so `/var/lib/postgresql` instead of `/var/lib/postgresql/data` appears to allow the container to start, however, I can still not login and it looks like it cannot read the database. Any help would be nice, I did not update my Postgresql instance, it is using a specific version number, so I am not sure why this happened.

**Comment by ZerkerEOD** [comment]
This is an example of the errors I am getting. 

```
2023-05-29 20:02:59.319 BST [72] ERROR:  relation "authentik_core_source" does not exist at character 533
2023-05-29 20:02:59.319 BST [72] STATEMENT:  SELECT "authentik_policies_policybindingmodel"."pbm_uuid", "authentik_policies_policybindingmodel"."policy_engine_mode", "authentik_core_source"."policybindingmodel_ptr_id", "authentik_core_source"."managed", "authentik_core_source"."name", "authentik_core_source"."slug", "authentik_core_source"."user_path_template", "authentik_core_source"."enabled", "authentik_core_source"."icon", "authentik_core_source"."authentication_flow_id", "authentik_core_source"."enrollment_flow_id", "authentik_core_source"."user_matching_mode" FROM "authentik_core_source" INNER JOIN "authentik_policies_policybindingmodel" ON ("authentik_core_source"."policybindingmodel_ptr_id" = "authentik_policies_policybindingmodel"."pbm_uuid") WHERE "authentik_core_source"."managed" = 'goauthentik.io/sources/inbuilt' LIMIT 21 FOR UPDATE
```

**Comment by itsOnly1Jah** [comment]
> This is an example of the errors I am getting.
> 
> ```
> 2023-05-29 20:02:59.319 BST [72] ERROR:  relation "authentik_core_source" does not exist at character 533
> 2023-05-29 20:02:59.319 BST [72] STATEMENT:  SELECT "authentik_policies_policybindingmodel"."pbm_uuid", "authentik_policies_policybindingmodel"."policy_engine_mode", "authentik_core_source"."policybindingmodel_ptr_id", "authentik_core_source"."managed", "authentik_core_source"."name", "authentik_core_source"."slug", "authentik_core_source"."user_path_template", "authentik_core_source"."enabled", "authentik_core_source"."icon", "authentik_core_source"."authentication_flow_id", "authentik_core_source"."enrollment_flow_id", "authentik_core_source"."user_matching_mode" FROM "authentik_core_source" INNER JOIN "authentik_policies_policybindingmodel" ON ("authentik_core_source"."policybindingmodel_ptr_id" = "authentik_policies_policybindingmodel"."pbm_uuid") WHERE "authentik_core_source"."managed" = 'goauthentik.io/sources/inbuilt' LIMIT 21 FOR UPDATE
> ```

I'm experiencing similar issued did you figure this out?

**Comment by itsOnly1Jah** [comment]
I figured it out. The dump file I was using to import into postgresql was not correct (different file sizes). I already copied the `dump.sql` to my local machine prior to upgrading so I just uploaded my local copy of the database back to the container.  
After doing so, I removed the `/bitnami/postgresql/data` directory which caused the pod to crash. Once the pod restarted I  re-imported the `dump.sql` with the following command `psql -U postgres -d $POSTGRES_DB -f dump.sql`
